### PR TITLE
Fixed EMM-1043 issue

### PR DESCRIPTION
**Pull request by tharinda221**

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/units/user-create/public/js/bottomJs.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ $( "#userStore" )
                 }
             }
         );
-    }).change();
+    })
+    .change();
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/units/user-create/public/js/bottomJs.js]: Please fix the formatting issues.

Comment (tharinda221) [inline modules/apps/jaggery/emm/src/emm/units/user-create/public/js/bottomJs.js]: Fixed

File: modules/apps/jaggery/emm/src/emm/modules/enrollments/ios/agent-controller.jag [hunk 2]
```diff
@@ -66,6 +66,8 @@ if (platform != "iOS") {
                 // one visible possibility would be server sending 500
                 response.sendRedirect(mdmProps["appContext"] + "enrollments/ios/login-agent?error=unexpected");
             }
+        }else{
```
Comment (milanperera) [inline modules/apps/jaggery/emm/src/emm/modules/enrollments/ios/agent-controller.jag]: It would be good if we can fix the formatting issue here.

Comment (tharinda221) [inline modules/apps/jaggery/emm/src/emm/modules/enrollments/ios/agent-controller.jag]: fixed

Comment (tharinda221) [inline modules/apps/jaggery/emm/src/emm/modules/enrollments/ios/agent-controller.jag]: https://github.com/wso2/product-mdm/pull/110

**Pull request merged**